openslides/core/views.py

```diff
@@ -265,10 +284,25 @@ def determine_metadata(self, request, view):
 
 class ConfigViewSet(ViewSet):
     """
-    API endpoint to list, retrieve and update the config.
+    API endpoint for the config.
+
+    There are the following views: list, retrieve and update.
     """
     metadata_class = ConfigMetadata
 
+    def check_view_permissions(self):
+        """
+        Returns True if the user has required permissions.
+        """
+        if self.action in ('list', 'retrieve'):
+            # Everybody is able to list and retrieve the config.
+            result = True
```

Comment:
When anonymous user is deactivated, then this function should return False for anonymous user.